File: result/translate/starchat/java2python/python/PROGRAM_CENSOR_WORD_ASTERISKS_SENTENCE.py
def f_gold(text, word):
        word_list = text.split(" ")
        result = ""
        stars = ""
        for i in range(len(word)):
            stars += "*"
        for i, word_i in enumerate(word_list):
            if word_i == word:
                word_list[i] = stars
        for word_i in word_list:
            result += word_i + " "
        return result

File: result/translate/starchat/python2java/java/PROGRAM_CENSOR_WORD_ASTERISKS_SENTENCE.java
public static String f_gold ( String text, String word ) {
        String[] word_list = text.split(" ");
        String result = "";
        int stars = word.length();
        for (int i = 0; i < word_list.length; i++) {
            if (word_list[i].equals(word)) {
                result = result + ("*" + word + "*");
            } else {
                result = result + word_list[i] + " ";
            }
        }
        return result;
    }